lib/gh/client.ml

```diff
@@ -2,6 +2,7 @@ module Json = Yojson.Basic.Util
 
 type error =
   | No_github_token
+  | Bad_credentials
```

Comment:
My only feedback: let's create the `bad_credentials` error type and put the message, documentation url and status code inside this type and decode it from JSON:

```ocaml
"{\"message\":\"Bad credentials\",\"documentation_url\":\"https://docs.github.com/graphql\",\"status\":\"401\"}"
```

And then display this information in the UI. Let's try to avoid loosing potentially useful debugging information.

Also, I'm not sure if "Bad credentials" is the only possible message. At least, if we display the message, we'll be able to learn about different error scenarios